topologies/datacenter-latest/labguides/source/mlag-with-cvp.rst

```diff
@@ -0,0 +1,116 @@
+MLAG
+====
+
+.. image:: images/cvp-mlag/mlag_1.png
+   :align: center
+
+.. note:: Did you know the “mlag” script is composed of Python code that
+          uses the CloudVision Portal REST API to automate the provisioning of
+          CVP Configlets. The configlets that are configured via the REST API
+          are ``Spine1-MLAG-Lab``, ``Spine2-MLAG-Lab``, ``Leaf1-MLAG-Lab``,
+          ``Leaf2-MLAG-Lab``, ``Leaf3-MLAG-Lab``. In addition each switch also
+          gets the ``VLANs`` configlet.
+
+.. note:: The manually-entered commands below that are part of this lab are
```

Comment:
@jrecchia1029 Not sure if we want to keep this or not, but if we do, maybe reference that the commands are equivalent to the Lea4-MLAG-Labl configlet in CVP.